### PR TITLE
Copy integrations repository content

### PR DESCRIPTION
**Pull request by Swiddis**
### Description
Companion PR to #552, this PR copies over the actual integration content. When this PR is merged, further updates to the integration content should be directed to this repo.

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.1.json
Comment (YANG-DB) [inline server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.1.json:1]: why do we need the second nginx json ?

File: server/adaptors/integrations/__data__/repository/nginx/schemas/communication-1.0.0.mapping.json
Comment (YANG-DB) [inline server/adaptors/integrations/__data__/repository/nginx/schemas/communication-1.0.0.mapping.json:1]: verify this match https://github.com/opensearch-project/opensearch-catalog/blob/main/schema/observability/logs/communication.mapping

File: server/adaptors/integrations/__data__/repository/nginx/schemas/http-1.0.0.mapping.json
Comment (YANG-DB) [inline server/adaptors/integrations/__data__/repository/nginx/schemas/http-1.0.0.mapping.json:1]: verify this match https://github.com/opensearch-project/opensearch-catalog/blob/main/schema/observability/logs/http.mapping

File: server/adaptors/integrations/__data__/repository/nginx/schemas/logs-1.0.0.mapping.json
Comment (YANG-DB) [inline server/adaptors/integrations/__data__/repository/nginx/schemas/logs-1.0.0.mapping.json:1]: verify this match https://github.com/opensearch-project/opensearch-catalog/blob/main/schema/observability/logs/logs.mapping

**Review by YANG-DB — APPROVED** [review]

File: server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.1.json
Comment (Swiddis) [reply server/adaptors/integrations/__data__/repository/nginx/nginx-1.0.1.json:1]: It's a new version, updated dashboard. Older dashboard has other panels.

**Review by YANG-DB — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**